tools/docgen/docgen/manifest.yaml

```diff
@@ -1,4 +1,35 @@
 channels:
-  - name: "channels.WebsiteChannel"
-    input_dir: "../../../docs/"
-    output_dir: "../../../website/docs/"
+#  - name: channels.WebsiteChannel
```

Comment:
Remove commented out website channel

Reply:
I made a mistake there. WebsiteChannel shouldn't be commented out. It will break website auto deployment.

Reply:
Sent you fix by teams